tests/providers/apache/livy/hooks/test_livy.py

```diff
@@ -316,6 +316,20 @@ def test_invalid_uri(self):
         with pytest.raises(RequestException):
             hook.post_batch(file='sparkapp')
 
+    @requests_mock.mock()
+    def test_extra_headers(self, mock):
+        mock.register_uri(
+            'POST',
+            '//livy:8998/batches',
+            json={'id': BATCH_ID, 'state': BatchState.STARTING.value, 'log': []},
+            status_code=201,
+            request_headers={'X-Requested-By': 'user'}
+        )
+
+        hook = LivyHook(extra_headers={'X-Requested-By': 'user'})
+        hook.post_batch(file='sparkapp')
+
```

Comment:
What does this test check?

Reply:
It checks that `X-Requested-By: user` is present in the request headers.

Reply:
Tested it locally, works fine